akka-persistence/src/main/scala/akka/persistence/journal/ReplayFilter.scala

```diff
@@ -46,6 +53,10 @@ private[akka] class ReplayFilter(persistentActor: ActorRef, mode: ReplayFilter.M
   import JournalProtocol._
   import ReplayFilter.{ Warn, Fail, RepairByDiscardOld, Disabled }
 
+  // for binary compatibility
+  def this(persistentActor: ActorRef, mode: ReplayFilter.Mode,
+           windowSize: Int, maxOldWriters: Int) = this(persistentActor, mode, windowSize, maxOldWriters, debugEnabled = false)
+
```

Comment:
As Patrik argued we do not actually need this, I just removed all exceptions while I was at it because I think that there is also some value in keeping that list short. This is a matter that can be debated, of course.